tfhe/src/shortint/key_switching_key/mod.rs

```diff
@@ -102,9 +102,12 @@ impl KeySwitchingKey {
     ///     PARAM_KEYSWITCH_1_1_KS_PBS_TO_2_2_KS_PBS,
     /// );
     ///
-    /// let cipher = ck1.encrypt(1);
+    /// let plaintext = 1;
```

Comment:
that's the cleartext as it's not encoded yet

Reply:
Done